src/content/docs/en/reference/dev-toolbar-app-reference.mdx

```diff
@@ -171,6 +171,20 @@ This event can be sent to inform the user that the app requires attention.
 
 Indicates whether or not the app has a notification for the user. When `true`, the app icon will be highlighted using a red dot. Conversely, when `false`, the highlight will be removed. If this property is not specified, `true` will be assumed.
 
+#### `level: 'info' | 'warning' | 'error'`
+
+Indicates the level of the notification. This will be used to determine the color of the highlight on the app icon. If this property is not specified, `'error'` will be assumed.
```

Comment:
```suggestion
Indicates the level of the notification. This will be used to determine the color and shape (blue square, gold triangle, or dark pink circle) of the highlight on the app icon. If this property is not specified, `'error'` will be assumed.
```

Reply:
It's not impossible for the colors to change if Fred has thoughts on them (though they were approved by Nate), but I doubt it. That's fine with me! I'll just make a small change to the order